myia/impl_py.py

```diff
@@ -0,0 +1,160 @@
+"""Implementations for the debug VM."""
+
+
+from typing import Dict, Callable
+from copy import copy
+from myia import primops
+
+
+implementations: Dict[primops.Primitive, Callable] = {}
```

Comment:
Globals should be in capitals.

Reply:
I think that's for constants, not dictionaries?